specification/help/resource-manager/readme.typescript.md

```diff
@@ -6,8 +6,8 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 ``` yaml $(typescript)
 typescript:
   azure-arm: true
-  package-name: "@azure/arm-help"
```

Comment:
has this name been reviewed by architects ?

Reply:
Yes it is reviewed by the architects. https://github.com/Azure/azure-sdk/issues/5817